### PR TITLE
E2E Enhancement Draft

### PR DESCRIPTION
**Pull request by noobwei**
**What type of PR is this?**

/kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: test/e2e/baseline_test.go [hunk 1]
```diff
@@ -909,3 +910,37 @@ func PromDiff(t test.Failer, prom prometheus.Instance, cluster cluster.Cluster,
 
 	}
 }
+
+func TestServiceRestart(t *testing.T) {
```
Comment (YaoZengzeng) [inline test/e2e/baseline_test.go:914]: This case fails and needs to be fixed.

Comment (hzxuzhonghu) [reply test/e2e/baseline_test.go:914]: donot we already have a restart case?

Comment (YaoZengzeng) [reply test/e2e/baseline_test.go:914]: > donot we already have a restart case?

We have `Kmesh daemon restart` case and this is the restart of applications.

File: test/e2e/baseline_test.go [hunk 2]
```diff
@@ -909,3 +910,37 @@ func PromDiff(t test.Failer, prom prometheus.Instance, cluster cluster.Cluster,
...
+						Port: echo.Port{
+							Name: "http",
+						},
+						Retry: echo.Retry{NoRetry: true},
+					},
+					Interval: callInterval,
+				}).Start()
+				generators = append(generators, g)
+			}
+			mkGen(dst)
```
Comment (YaoZengzeng) [inline test/e2e/baseline_test.go:937]:
```suggestion
			apps.ServiceWithWaypointAtServiceGranularity[0]
```

Not use `dst` itself as `src`.

File: test/e2e/baseline_test.go [hunk 3]
```diff
@@ -909,3 +910,37 @@ func PromDiff(t test.Failer, prom prometheus.Instance, cluster cluster.Cluster,
...
+					Source: src,
+					Options: echo.CallOptions{
+						To:    dst,
+						Count: 1,
+						Check: check.OK(),
+						HTTP:  echo.HTTP{Path: "/?delay=10ms"},
+						Port: echo.Port{
+							Name: "http",
+						},
+						Retry: echo.Retry{NoRetry: true},
```
Comment (YaoZengzeng) [inline test/e2e/baseline_test.go:931]: Remove this line, as it takes time to send new endpoints to eBPF. So we need retry.

**Pull request merged**